tracer/src/Datadog.Trace.Trimming/build/Datadog.Trace.Trimming.xml

```diff
@@ -0,0 +1,956 @@
+<linker>
+   <assembly fullname="AerospikeClient" />
```

Comment:
This file should change everytime we reference a new assembly or type. So I think _either_

- We exclude this file from the repo
- We add a check to the build that this file hasn't changed, similar to what we do for source generators, so that we know the file in the repo is the one we're packing into the package

Reply:
> - We add a check to the build that this file hasn't changed, similar to what we do for source generators, so that we know the file in the repo is the one we're packing into the package

I like this approach a lot

Reply:
done in [f672201](https://github.com/DataDog/dd-trace-dotnet/pull/4195/commits/f672201b3d15289bb61aab4d63259bf8da504e8b)